app/views/users/settings.html.slim

```diff
@@ -2,36 +2,4 @@
 - content_for :page_heading, "Settings"
 
 .row
-  .col-xs-3
-    | Word Cloud
-  .col-xs-2
-    input#word-cloud-switch type="checkbox"
-
-.row
-  .col-xs-3
-    | Doge Cloud
-  .col-xs-2
-    input#doge-switch type="checkbox"
-
-
-javascript:
-
-  $(document).ready(function() {
-    var word_cloud = #{@word_cloud};
-    var doge = #{@doge};
-
-    if (!word_cloud)
-    {
-      $("#doge-switch").bootstrapSwitch('disabled', true);
-    }
-
-    else
-    {
-      $("#word-cloud-switch").bootstrapSwitch('state', true, true);
-
-      if (doge)
-      {
-        $("#doge-switch").bootstrapSwitch('state', true, true);
-      }
-    }
-  });
```

Comment:
Should probably also remove the controller methods for this in the users controller